internal/buildscripts/packaging/msi/make.ps1

```diff
@@ -65,10 +65,8 @@ function Confirm-MSI {
     # start service
     Start-Service otelcol
 
-    # uninstall msi, validate service is uninstalled
+    # uninstall msi
     Start-Process -Wait msiexec "/x `"$msipath`" /qn"
-    sc.exe query state=all | findstr "otelcol" | Out-Null
-    if ($LASTEXITCODE -ne 1) { Throw "otelcol service failed to uninstall" }
```

Comment:
maybe provide a reason/justification for upstream on why we're removing this

Reply:
This test relies on the query failing with exit code 1 due to the service being uninstalled and then handling the exit code. On GitHub Actions, exit code 1 fails the step and the workflow. The simplest solution would be to remove validating the uninstall in this bizarre way.